tools/mksurfdata_map/mksurfdata_map.namelist

```diff
@@ -24,7 +24,8 @@
  map_fch4         = '/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/maps/10x15/map_360x720_cruncep_to_10x15_nomask_aave_da_c130326.nc'
  mksrf_fsoitex    = '/glade/p/cesm/cseg/inputdata/lnd/clm2/rawdata/mksrf_soitex.10level.c010119.nc'
  mksrf_forganic   = '/glade/p/cesm/cseg/inputdata/lnd/clm2/rawdata/mksrf_organic_10level_5x5min_ISRIC-WISE-NCSCD_nlev7_c120830.nc'
- mksrf_flakwat    = '/glade/p/cesm/cseg/inputdata/lnd/clm2/rawdata/mksrf_LakePnDepth_3x3min_simyr2004_csplk_c151015.nc'
+ mksrf_flakwat    = '/glade/u/home/ivanderk/clm5.0/own_inputfile/mksurf_LakePnDepth_3x3min_simyr2009_csplk_c190603.nc
+'
```

Comment:
There's extra whitespace at the end of the filename and onto the next line, that should be removed.

Reply:
This is removed in 0a38f6d7fc1ab68ce5f0845da6a1a64b8724b2bd.